resources/qml/SaveButton.qml

```diff
@@ -48,14 +48,15 @@ Item {
     }
 
     function sliceOrStopSlicing() {
-        try {
-            if ([1, 5].indexOf(base.backendState) != -1) {
-                CuraApplication.backend.forceSlice();
-            } else {
-                CuraApplication.backend.stopSlicing();
-            }
-        } catch (e) {
-            console.log('Could not start or stop slicing', e)
+        if ( [ 1, 5 ].indexOf( base.backendState ) != -1 )
+        {
+            prepareButton.preparingToSlice = true;
+            CuraApplication.backend.forceSlice();
+        }
+        else
+        {
+            prepareButton.preparingToSlice = false;
+            CuraApplication.backend.stopSlicing();
```

Comment:
Why was the try/catch statement removed? That was probably added for a reason, to catch special situations.

Reply:
Chris added it and we don't do that anywhere else nor were we really handling the errors. This logic should be done on the Python side and use the proper logger class.